src/tools/wasm-opt.cpp

```diff
@@ -240,6 +240,7 @@ int main(int argc, const char* argv[]) {
     auto input = buffer.getAsChars();
     WasmBinaryBuilder parser(other, input, false);
     parser.read();
+    options.applyFeatures(other);
```

Comment:
why is this needed / how did things work without it before? :)

Reply:
This is in the fuzzing code, so I guess we hadn't tried fuzzing with features recently.